pkg/obsservice/obslib/migrations/sqlmigrations/0001_init.sql

```diff
@@ -1,5 +1,13 @@
 -- +goose Up
-CREATE TABLE events();
+CREATE TABLE cluster_events(
+  timestamp TIMESTAMP NOT NULL,
+  id BYTES NOT NULL DEFAULT uuid_v4(),
+  cluster_id BYTES NOT NULL,
+  node_id INT NOT NULL,
+  event_type STRING NOT NULL,
+  event JSONB,
+  CONSTRAINT "primary" PRIMARY KEY (timestamp, id) USING HASH WITH BUCKET_COUNT = 16
```

Comment:
These days you don't need the `WITH BUCKET_COUNT = 16`, it's the default. Also, this syntax is deprecated in favor of `CONSTRAINT "primary" PRIMARY KEY (timestamp, id) USING HASH WITH (bucket_count = 16)` if you did want to state it explicitly.